pkg/cmd/provisioning/azure/delete.go

```diff
@@ -80,6 +81,14 @@ func deleteManagedIdentities(client *azureclients.AzureClientWrapper, name, reso
 	for listManagedIdentities.More() {
 		pageResponse, err := listManagedIdentities.NextPage(context.Background())
 		if err != nil {
+			var respErr *azcore.ResponseError
+			if !(errors.As(err, &respErr)) {
+				return err
+			}
+			if respErr.ErrorCode == "ResourceGroupNotFound" {
+				log.Printf("Found no user-assigned managed identities in resource group %s", resourceGroupName)
```

Comment:
Prefer if this said something about the resource group already being gone / having been deleted. Something like this similar to the latter `"Found no resource group %s"` log.

```suggestion
				log.Printf("Found no resource group %s. No user-assigned managed identities to list", resourceGroupName)
```

Reply:
[Done](https://github.com/openshift/cloud-credential-operator/compare/cc1d63226f953c4ada6539888746cf4cbe463e1f..f83c6b70da024ca5e159d2515ab149227bad58d9)